certified-connectors/Pilot Things/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5915 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This documentation expose the principals endpoints of the Pilot Things api. For any additional details or question see the support page : \nhttps://support.pilot-things.com/hc/en-ca",
```

Comment:
Could this description be modified to more reflect the functionality the connector offers and the value it provides to users? This will be one of the first things that a user might see with the connector to learn more about it, so it's a good place to discuss what it brings to the table.

Reply:
For sure! I just changed it to the same thing as the first line of the README.